### DISCUSSION TITLE
identity Showing Unhealthy

### DISCUSSION THREAD

**Discussion by lawallanre00490038** [Sunbird-RC/community#1050] [Issues]
When I checked the logs  docker logs new_sunbird-identity-1, I found,

lawal@David:~/new_sunbird$ docker logs new_sunbird-identity-1

> did-l3-ms@1.0.0 start:migrate:prod
> npx prisma migrate deploy && node dist/main

Prisma schema loaded from prisma/schema.prisma
Datasource "db": PostgreSQL database "registry", schema "public" at "db:5432"

6 migrations found in prisma/migrations

Error: P3005

The database schema is not empty. Read more about how to baseline an existing production database: https://pris.ly/d/migrate-baseline

npm notice
npm notice New minor version of npm available! 10.2.4 -> 10.8.2
npm notice Changelog: <https://github.com/npm/cli/releases/tag/v10.8.2>
npm notice Run `npm install -g npm@10.8.2` to update!
npm notice

Please how do I solve this??

**Comment by pallakartheekreddy** [comment]
@lawallanre00490038 Please let us know which version of sunbird-rc you are trying to install and can you provide more logs to help on this issue.

Check below thread, it might be helpful
https://github.com/orgs/Sunbird-RC/discussions/967
